modules/80_optimization/nash/equations.gms

```diff
@@ -13,7 +13,7 @@ q80_budg_intertemp(regi)..
   + SUM(ttot$(ttot.val ge 2005),
      pm_ts(ttot)
       * (
-        SUM(trade$(NOT tradeSe(trade)),
+        SUM(trade$(NOT tradeSe(trade) and NOT tradeCap(trade)),
```

Comment:
is it correct that the new tradeCap-set is not included in the Nash convergence?

Reply:
First of all, it doesn't matter for anyone except me, as the `tradeCap` set is empty in all realisations other than `capacity`. So any normal user working with `standard` trade will be unaffected by this change.

As an explanation why this is removed: it did not have to be there in the first place, because currently the `trade` set (which is calculated automatically by REMIND) simply doesn't contain `tradeCap`. The question whether in the long run `tradeCap` should be part of `trade` is one that needs to be determined. Renato opted to include `tradeSe` in `trade` but then exclude it explicitly in a few places. I have so far not done this but instead explicitly added `tradeCap` in any sum where it was needed.